File: androidanimation06/app/src/main/java/com/clwater/animation/MainActivity.java
package com.clwater.animation;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        CustomSunView cv_main = findViewById(R.id.cv_main);
        cv_main.setText("测试");
        cv_main.start();
    }
}

File: androidanimation03/src/main/java/clwater/androidanimation03/MainActivity.kt
package clwater.androidanimation03

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        animationplanet.setOnClickListener {
            animationplanet.changeView()
        }

    }
}
